frontend/actions/siteActions.js

```diff
@@ -83,7 +83,9 @@ export default {
         });
     }
 
-    return onRemoveUser.then(onUserRemoveFromSite);
+    return onRemoveUser
+      .then(userActions.fetchUser)
+      .then(onUserRemoveFromSite);
```

Comment:
Hrm, so on the Users view, I clicked Remove User to remove a user from my site. The User Action table refreshed and showed the new action in it, but the table of users did not refresh (so the removed user was still listed). Maybe the site also needs to be refreshed at this point?

I should have made a gif, but here's a screenshot of the "bad" state showing both the user that I removed and the action that removed the user:

<img width="1045" alt="screen shot 2018-01-02 at 3 16 23 pm" src="https://user-images.githubusercontent.com/697848/34501564-114c6f16-efd5-11e7-8c9c-51b2e3584352.png">

Reply:
Innnnnteresting. I thought it was working properly when I opened the PR but I don't have a GIF to prove it! I'll check this out.